### DISCUSSION TITLE
Reusing data in another component

### DISCUSSION THREAD

**Discussion by geovla93** [vercel/swr#1280] [Q&A] [answered]
Hello everyone. I want to make the same request in two different components to avoid prop drilling. Will it make 2 separate requests? Or just one and then the second will get the data from cache?

**Comment by shuding** [comment, marked as answer]
Please read the documentation here: https://swr.vercel.app/docs/getting-started#make-it-reusable.

**Reply by geovla93** [reply]
Thank you so much exactly what I needed!
